honeybee_energy/constructionset.py

```diff
@@ -1314,21 +1390,23 @@ class DoorConstructionSet(object):
         * constructions
         * modified_constructions
         * is_modified
+        * user_data
     """
     __slots__ = ('_exterior_construction', '_interior_construction',
                  '_exterior_glass_construction', '_interior_glass_construction',
-                 '_overhead_construction', '_locked')
+                 '_overhead_construction', '_locked', '_user_data')
 
     def __init__(self, exterior_construction=None, interior_construction=None,
                  exterior_glass_construction=None, interior_glass_construction=None,
-                 overhead_construction=None):
+                 overhead_construction=None, _user_data=None):
```

Comment:
The `_user_data=None` should be removed from the `__init__` arguments. This keeps things consistent with the other classes and you aren't using this argument now.

Reply:
commit: constructionSet init args

Yeahh.. I'm not sure how I ended up doing that for some reason lol